torch/ao/quantization/fx/prepare.py

```diff
@@ -1098,30 +1072,27 @@ def insert_observers_for_model(
     passes which optimize the graph by deduplicating observers, etc.
     """
 
-    # name of Node in original FX Graph to the target dtype information
+    # node.meta["target_dtype_info"] stores the target dtype information
     # that's derived from qconfig for the Node, for example, if we have
     # a conv2d node that has a qconfig
     # {
     #   # information for input and bias node omitted
     #   # for getattr node
     #   # weight = getattr(self, 'weight')
-    #   'weight': {
-    #      'output_activation_dtype': torch.float,
+    #   weight_node.meta["target_dtype_info"] = {
+    #      'output_activation_dtype': (torch.float, False)
     #   }
+    #   # Note: False means it's not a dynamic quantization (but a static quantization)
     #   # for conv2d node
     #   # conv2d = call_function[target=torch.nn.functional.conv2d](
     #   #            args=(input, weight, bias))
-    #   'conv2d': {
-    #       'input_activation_dtype': torch.quint8,
-    #       'weight_dtype': torch.qint8,
-    #       'bias_dtype': torch.float,
-    #       'output_activation_dtype': torch.quint8,
-    #     }
+    #   conv2d_node.meta["target_dtype_info"] = {
```

Comment:
nit: should this be `conv2d` here and also `weight` above (L1082)? or maybe change `conv2d` to `conv2d_node` in above line.

Reply:
sure